tendrl/monitoring_integration/upgrades/utils.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+
+def print_message(message):
```

Comment:
Do we really need a utility function for printing message :)

Reply:
i saw messages are formatted with \n every where, that why i put print in utils, i think we don't need, i will revert it back